.github/workflows/ci.yml

```diff
@@ -83,15 +80,11 @@ jobs:
           ]
         exclude:
           # PyPy doesn't release 32-bit Windows builds any more
-          - python-version: pypy-3.7
-            platform: { os: "windows-latest", python-architecture: "x86" }
-          - python-version: pypy-3.8
-            platform: { os: "windows-latest", python-architecture: "x86" }
           - python-version: pypy-3.9
             platform: { os: "windows-latest", python-architecture: "x86" }
         include:
           # Test minimal supported Rust version
-          - rust: 1.63.0
+          - rust: 1.70.0
```

Comment:
Good question on the MSRV bump. In PyO3 we support ranges of optional dependencies and have a helper script to downgrade them for the MSRV job to compatible versions. I'm unsure what the best option is here. Keeping the MSRV down is useful for linux distributions putting these packages in their package managers, e.g. https://github.com/PyO3/pyo3/issues/3113

We last did an MSRV bump with PyO3 0.22 to 1.63, which is the current version supported by Debian stable.

Reply:
Basing the MSRV on Debian stable and aligning it with PyO3's requirements, rather than targeting the latest version required by `tokio`, feel like the right approach (I'll revert back to 1.63). I'm just not sure of the best way to set this up in CI to avoid the error with `tokio`. The helper script sounds like what we need, but I'll need some time to figure it out.

Reply:
Yep, I can perhaps try to port that script tomorrow 👍